gcc/0951-11.2.0-msys2-spec.patch

```diff
@@ -287,11 +296,14 @@ diff -Naur gcc-10.2.0-orig/gcc/config.gcc gcc-10.2.0/gcc/config.gcc
 +	extra_objs="${extra_objs} winnt.o winnt-stubs.o"
 +	c_target_objs="${c_target_objs} msformat-c.o"
 +	cxx_target_objs="${cxx_target_objs} winnt-cxx.o msformat-c.o"
++	d_target_objs="${d_target_objs} winnt-d.o"
++	target_has_targetdm="yes"
 +	if test x$enable_threads = xyes; then
 +		thread_file='posix'
 +	fi
 +	default_use_cxa_atexit=yes
 +	use_gcc_stdint=wrap
++	default_use_cxa_atexit=yes
```

Comment:
Probably a copy-paste situation, but this line seems superfluous.

Reply:
that patch is from cygwin: https://cygwin.com/git-cygwin-packages/?p=git/cygwin-packages/gcc.git;a=blob;f=0007-Cygwin-__cxa-atexit.patch;h=6933294e24832fb368cf039df8a34ac0d8c2b43c;hb=HEAD

I agree, but maybe we just leave it to avoid a diff to cygwin(?)